source/_integrations/comfoair.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: "Zehnder ComfoAir Ventilation"
+description: "Instructions on how to integrate Zehnder ComfoAir 350 ventilation systems into Home Assistant."
+logo: zehnder.png
+ha_category:
+  - Fan
+  - Sensor
+ha_release: 0.101
+---
+
+The `comfoair` integration lets you control Zehnder ComfoAir [350](https://www.international.zehnder-systems.com/products-and-systems/comfosystems/zehnder-comfoair-350) ventilation units from Home Assistant. A serial connection is needed to connect the unit to Home Assistant.
+
+The integration has a fan platform to view and control the ventilation speed, and a sensors platform to read out various temperatures. It passively listens to messages exchanged between CC Ease control units and the system board. It can take control of the system temporarily to set fan speeds.
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+comfoair:
```

Comment:
We would like to see only required config variables in the first example

Reply:
Fixed.